lib/morse_translator.rb

```diff
@@ -1,6 +1,8 @@
 class MorseTranslator
+  attr_reader :string
 
   def initialize
+    @translated_string = []
```

Comment:
Instead of making an instance variable, you should make a local variable inside your translate method